### ISSUE TITLE
Images in LSX blog posts are only display at full size

### ISSUE THREAD

**Issue by antonvandiermen** [lightspeeddevelopment/lsx#33]
This is the post where the issue was experienced. The images have now been removed and replaced with text links to the payment gateways - https://www.lsdev.biz/wp-admin/post.php?post=13692&action=edit

The images looked like this, but have now been removed:
![sa payment gateways for woocommerce](https://cloud.githubusercontent.com/assets/14092353/9758401/b05ea284-56e6-11e5-910f-66d6346abd0b.png)

The HTML for the page was as follows:
<PLAINTEXT>
<table style="height: 150px;" width="390">
<tbody>
<tr>
<td width="534">
<h6><strong>For South African PayPal enquiries please use the following details:</strong></h6>
</td>
</tr>
<tr>
<td><strong>Call</strong> - 0861 PAYPAL (729 725)</td>
</tr>
<tr>
<td><strong>Email</strong> - paypalenquiries@fnb.co.za</td>
</tr>
<tr>
<td><strong>Fax</strong> - 011 263 5139</td>
</tr>
<tr>
<td><strong>For PayPal specific queries</strong> - http://www.paypal.com/za</td>
</tr>
</tbody>
</table>
<span style="font-size: 14px;">This marks the start in a new series we will be writing about, the focus is on SA Payment Gateways that have extensions available for your <a title="WooCommerce" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/woocommerce/" target="_blank">WooCommerce</a> powered shop. Setting up you online shop can be a daunting task. Between the developers, the payment gateway and the bank, there is a lot to keep track of. We hope that this series will clearly explain the processes involved. </span>

Today we lay the groundwork for this series. After reading, you should have a firm grasp on the services available, their costs, and have a clear understanding about how the all work together in you WooCommerce store.
<h2>What is a Payment Gateway and how does it work?</h2>
A Payment Gateway is an e-commerce service provider. Payment gateways protect credit card details by encrypting sensitive information to ensure that information is passed securely between the customer and the merchant and also between the merchant and the bank's payment processor. It facilitates the transfer of information between a website and the acquiring bank account.

Warning. This list is heavy on jargon, but some of you might be interested in the details
<ol>
	<li>A customer places an order on the website using their preferred payment method after entering their details, usually credit card information.</li>
	<li>The customer's web browser encrypts the information to be sent between the browser and the merchant's webserver. This is done via SSL (Secure Socket Layer) encryption.</li>
	<li>The merchant then forwards the transaction details (automated) to their payment gateway. This is another SSL encrypted connection to the payment server hosted by the payment gateway.</li>
	<li>The payment gateway forwards the transaction information to the payment processor used by the merchant's acquiring bank.</li>
	<li>The payment processor forwards the transaction information to the card association (e.g., Visa/MasterCard)</li>
	<li>The credit card issuing bank receives the authorization request and does fraud and credit or debit checks and then sends a response back to the processor with a response code (eg: approved, denied). The response code is used to define the reason why the transaction failed (such as insufficient funds, or bank link not available). Meanwhile, the credit card issuer holds an authorization associated with that merchant and consumer for the approved amount. This can impact the consumer's ability to further spend (eg: because it reduces the line of credit available or because it puts a hold on a portion of the funds in a debit account).</li>
	<li>The processor forwards the authorization response to the payment gateway.</li>
	<li>The payment gateway receives the response, and forwards it on to the website where it is interpreted as a relevant response then relayed back to the merchant and cardholder. This is known as the Authorization or "Auth"</li>
	<li>The process usually takes 2 to 3 seconds.</li>
	<li>The merchant then fulfills the order and the above process is repeated but this time to "Clear" the authorization by consummating the transaction. Typically the "Clear" is initiated only after the merchant has fulfilled the transaction (eg: shipped the order). This results in the issuing bank 'clearing' the 'auth' (ie: moves auth-hold to a debit) and prepares them to settle with the merchant acquiring bank.</li>
	<li>The merchant submits all their approved authorizations, in a "batch" (eg: end of day), to their acquiring bank for settlement via its processor.</li>
	<li>The acquiring bank makes the batch settlement request of the credit card issuer.</li>
	<li>The credit card issuer makes a settlement payment to the acquiring bank (eg: the next day)</li>
	<li>The acquiring bank subsequently deposits the total of the approved funds in to the merchant's nominated account (eg: the day after). This could be an account with the acquiring bank if the merchant does their banking with the same bank, or an account with another bank.</li>
</ol>
<h2>SA Payment Gateway Fees - Supported in WooCommerce</h2>
These payment gateways all have extensions available for WooCommerce. The benefit of using the extensions is that payment gateways are setup and configured, rather than developed into your website, that would be the traditional development route. But if you use WordPress and have WooCommerce, you can purchase one of these payment gateway extensions, configure a few options and start selling in minutes. Well almost, assuming you have already opened a merchant account with one of the major South African banks and have signed up with a payment gateway. The only exception is WebCash, they don't require you to have a merchant account.

Click the logos below for each payment gateway's website.

<center><a href="http://mygate.co.za/" target="_blank"><img class="wp-image-13760" style="padding: 20px;" title="Mygate" src="https://www.lsdev.biz/wp-content/uploads/2012/07/Mygate.jpg" alt="" /></a><a href="https://www.payfast.co.za/" target="_blank"><img class="wp-image-13761" style="padding: 20px;" title="PayFast" src="https://www.lsdev.biz/wp-content/uploads/2012/07/PayFast.jpg" alt="" /></a><a href="https://www.paygate.co.za/index.php" target="_blank"><img class="wp-image-13762" style="padding: 20px;" title="PayGate" src="https://www.lsdev.biz/wp-content/uploads/2012/07/PayGate.jpg" alt="" /></a></center><center><a href="https://www.vcs.co.za/" target="_blank"><img class="wp-image-13763" style="padding: 20px;" title="VCS" src="https://www.lsdev.biz/wp-content/uploads/2012/07/VCS.jpg" alt="" /></a><a href="https://www.vcs.co.za/" target="_blank"><img class="wp-image-13764" style="padding: 20px;" title="Webcash" src="https://www.lsdev.biz/wp-content/uploads/2012/07/Webcash.jpg" alt="" /></a></center>All these payment gateways are available for purchase from <a title="WooCommerce" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/woocommerce/" target="_blank">WooCommerce</a> for $50. <a title="MyGate Payment Gateway" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/extension/mygate-payment-gateway/" target="_blank">MyGate</a> • <a title="Virtual Card Services" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/extension/vcs/" target="_blank">Virtual Card Services</a> • <a title="WebCash" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/extension/webcash/" target="_blank">WebCash</a> • <a title="PayGate" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/extension/paygate/" target="_blank">PayGate</a> • <a title="PayFast" href="http://zfer.us/9Hkcv?d=http://www.woothemes.com/extension/payfast/" target="_blank">PayFast</a>
<h2>SA Payment Gateway Fees</h2>
These are only the gateways supported by WooCommerce or have an extension for WooCommerce. All Fees are VAT exclusive.

Merchant Account Fees

Where 'negotiable' is indicated, please know that some aspects of the pricing is entirely negotiated between the bank sales consultant and the applicant. The bank needs to assess your business in terms of risk, transaction values and turnover. If you are a new business, you will need to give projections, most likely for the first and second year. Existing business will need to do the same but also give past bank statements. All this information is found in the merchant account application forms issued by the banks. Contact your bank to find out about getting a merchant account.

From here on out we will discuss each payment gateway in a series of posts that describes the features available and how to use them in your WooCommerce powered shop. If you would like to follow this entire series, please sign up for our newsletter, to stay up to date with this and all other topics we discuss here on lsdev.biz.
<h2>Newsletter Signup</h2>

<div id="mc_embed_signup"><form id="mc-embedded-subscribe-form" class="validate" action="http://wordcamp.us2.list-manage2.com/subscribe/post?u=e50b2c5c82f4b42ea978af479&amp;id=156c94f56c" method="post" name="mc-embedded-subscribe-form" target="_blank"><input id="mce-EMAIL" class="email" name="EMAIL" type="email" value="" />
<div class="clear"><input id="mc-embedded-subscribe" class="button" name="subscribe" type="submit" value="Subscribe" /></div>
</form></div>

<h2>Terms</h2>
<ul>
	<li><strong>SSL:</strong> A protocol for transmitting private documents via the Internet.</li>
	<li><strong>Payment Processor:</strong> An entity in the credit card processing network that handles the posting of transactions for authorization, clearing and settlement to consumer credit card accounts at the card associations, and the settlement of funds to merchant bank accounts. Processors may also provide merchants with billing and reporting services.</li>
	<li><strong>Credit Card Issuer:</strong> The card association or branded payment cards directly to consumers.</li>
</ul>
<h2>Sources</h2>
<ul>
	<li><a title="webopedia - payment gateway" href="http://www.webopedia.com/TERM/P/payment_gateway.html" target="_blank">http://www.webopedia.com/TERM/P/payment_gateway.html</a></li>
	<li><a href="http://www.webopedia.com/TERM/P/payment_processing.html" target="_blank">http://www.webopedia.com/TERM/P/payment_processing.html</a></li>
	<li><a href="http://www.webopedia.com/TERM/S/SSL.html" target="_blank">http://www.webopedia.com/TERM/S/SSL.html</a></li>
	<li><a href="http://www.webopedia.com/TERM/M/merchant_account.html" target="_blank">http://www.webopedia.com/TERM/M/merchant_account.html</a></li>
	<li><a href="http://www.webopedia.com/quick_ref/ecommerce_shopping_process.asp" target="_blank">http://www.webopedia.com/quick_ref/ecommerce_shopping_process.asp</a></li>
	<li><a href="http://www.authorize.net/resources/howitworksdiagram/" target="_blank">http://www.authorize.net/resources/howitworksdiagram/</a></li>
	<li><a href="http://www.authorize.net/resources/glossary/" target="_blank">http://www.authorize.net/resources/glossary/</a></li>
	<li><a href="http://en.wikipedia.org/wiki/Payment_gateway" target="_blank">http://en.wikipedia.org/wiki/Payment_gateway</a></li>
</ul>

**Comment by krugazul** [comment]
@antonvandiermen the styles that are doing this have been removed from the shop branch.   The shop branch will be merged into the master branch on wednesday, all the fixes should carry through from there.

https://github.com/lightspeeddevelopment/lsx/commit/b6a85891d44829f1f130211561615032e29c0ae5
